py/mpconfig.h

```diff
@@ -1551,6 +1551,11 @@ typedef double mp_float_t;
 #define MP_UNLIKELY(x) __builtin_expect((x), 0)
 #endif
 
+#ifndef STRINGIFY
```

Comment:
py/misc.h already has `MP_STRINGIFY_HELPER`

Reply:
Yep, I found that all too late :(

Removed this commit and fixed code to use the `MP_STRINGIFY` macro.